src/main/java/network/brightspots/rcv/TabulatorSession.java

```diff
@@ -286,7 +286,9 @@ private List<CastVoteRecord> parseCastVoteRecords(ContestConfig config, Set<Stri
           reader.readCastVoteRecords(castVoteRecords, source.getContestId());
           // Before we tabulate, we output a converted generic CSV for the CVRs.
           try {
-            ResultsWriter writer = new ResultsWriter().setTimestampString(timestampString);
+            ResultsWriter writer = new ResultsWriter()
+                    .setContestConfig(config)
```

Comment:
I'm baffled by how this was working before without setting the config?!

Reply:
The function just never used `config` before, but now we use it to map display name to code. Per your other comment, we'll probably be reverting this if we get rid of `code`.

Reply:
Sounds good -- let's note / link to this comment in the issue / comment you make to remove code.

Reply:
https://github.com/BrightSpots/rcv/issues/663